index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <title>My First Portfolio</title>
+</head>
+<body>
+        <header class="welcome-message">
+            <a class="welcome">Welcome</a>
+            <a class="menu-icon"><i class="ri-menu-line"></i></a>
+        </header>
+        <section class="greetingsandsocial">
+            <h2 class="intro1">Hey there. I'm Edgard</h2>
+            <h2 class="intro2">I'm a software developer</h2>
+            <p class="intromessage"> I can help you build a product , feature or website. 
+            Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
```

Comment:
- [ ] [ Optional] You should stick to the design as much as possible. please use the same font in a paragraph. that is font-family: 'Inter';  Also there is more gap between icons please fix it
### Your design
![his](https://user-images.githubusercontent.com/81727988/212444966-95ed7566-7fc1-492c-b9a0-ce0dc6bbac34.jpg)


### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/212444972-cf660e72-1c61-46dc-9361-a1fcd81eff1c.jpg)

Reply:
@roshan-bajgain now I have fixed the font issue, I needed to add sans-serif font, but that is not in the template of figma. Also, I have changed the social media items, however I did not use the values in the figma template because they don't look the same when using them in my project. Can you please take another look before merging? Thanks

Reply:
@edgardsoza, It's good but you need to stick to the design in Figma. Thank you

Reply:
@edgardsoza, Hello bro please use this in html file <link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;300;600;800&display=swap" rel="stylesheet">
and use this font `font-family: 'Inter', sans-serif;`

Reply:
@edgardsoza, Import this and use it
<style>
  @import url('https://fonts.googleapis.com/css2?family=Inter:wght@200;300;600;800&display=swap');
</style>

This in you style file = font-family: 'Inter', sans-serif;

Reply:
I did already both bro!! Thanks so much. @roshan-bajgain